fuzz/fuzz.c

```diff
@@ -346,7 +355,9 @@ int main(int argc, char *argv[])
     __AFL_INIT();
     buf = __AFL_FUZZ_TESTCASE_BUF;
     while (__AFL_LOOP(100000)) {
-        len = __AFL_FUZZ_TESTCASE_LEN;
+        len = LIMIT_LEN(__AFL_FUZZ_TESTCASE_LEN);
```

Comment:
Is this macro defined somewhere in fuzzer headers? The other macros seem to start with double underscores, so this seems suspicious.

Reply:
It’s not defined anywhere, sorry. This is a leftover from an earlier draft of the patch which cut off excess bytes instead of skipping the testcase; apparently I forgot to remove it here and didn't build in AFL mode since.